### DISCUSSION TITLE
Nim based apps for Linux

### DISCUSSION THREAD

**Discussion by nandgator** [elementary/edw#52] [Ideas] [answered]
No sure if I can post more than one idea 🤔.

Nim is an imperative programming language. In one word I could be summed up as "elegant" in all manner. If possible I'd like to hear about creating Nim based application for Linux esp. keeping the elementary OS style guide / sheet.
 
Related Nim UI library: https://github.com/treeform/fidget

**Comment by JeysonFlores** [comment, marked as answer]
AppCenter just accept native GTK applications, that UI library does not work with GTK so the eOS stylesheet cannot be applied.

**Reply by nandgator** [reply]
Oh I see, Nim bindings for GTK3 are still in development https://github.com/StefanSalewski/oldgtk3. Maybe for another edw then. Thanks!

**Reply by nandgator** [reply]
Oh here's another: https://github.com/StefanSalewski/gintro

**Reply by JeysonFlores** [reply]
_Oh here's another: https://github.com/StefanSalewski/gintro_
These bindings are actually pretty good, even they're in a initial state if you're skilled enough it seems that you can do quite complex stuff. I took a look at them some time ago and I found a really solid work, however I will wait some time to see how the project keeps going.
